SUPPORT.md

```diff
@@ -0,0 +1,13 @@
+# Support
+
+## How to file issues and get help  
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing
+issues before filing new issues to avoid duplicates.  For new issues, file your bug or
+feature request as a new Issue.
+
+Need Support with Azure services? Every customer with an active Azure subscription has access to [support](https://docs.microsoft.com/azure/azure-supportability/how-to-create-azure-support-request) with guaranteed response times. Consider submitting a ticket and get assistance from the Microsoft support team.

```

Comment:
"Every customer with an active Azure subscription"... maybe basic non-paying users do not have access to file support tickets directly. Gotta check that.